src/navigation/NavigationContainer.tsx

```diff
@@ -0,0 +1,26 @@
+import { createStackNavigator, StackScreenProps } from '@react-navigation/stack';
+import { NavigationContainer as ReactNavigationContainer } from '@react-navigation/native';
+
+import MainPage from '../MainPage';
+import DetailsPage from '../DetailsPage';
```

Comment:
For some reason patch below break the project and show error "@/MainPage" cannot be found
```suggestion
import MainPage from '@/MainPage';
import DetailsPage from '@/DetailsPage';
```

Reply:
to make type alias you also have to take couple of additional steps

1. add [babel module resolver](https://www.npmjs.com/package/babel-plugin-module-resolver)
2. in `babel.config.js` or `.babelrc` add:
```
module.exports = {
  presets: ['module:@react-native/babel-preset'],
  plugins: [
    [
      'module-resolver',
      {
        root: ['./src'], // if you are setting root
        alias: {
        // your alias here probably "@": "./src"
        },
        extensions: ['.js', '.jsx', '.ts', '.tsx'],
      },
    ],
  ],
};

```

Reply:
Thanks you so much just, just tried and it work!